stl/src/vector_algorithms.cpp

```diff
@@ -2075,6 +2075,75 @@ namespace {
         }
         return _Result;
     }
+
+    template <class _Ty>
+    const void* __stdcall __std_find_first_of_trivial_impl(
+        const void* _First1, const void* const _Last1, const void* const _First2, const void* const _Last2) noexcept {
+#ifndef _M_ARM64EC
+        constexpr bool _Bytes = sizeof(_Ty) == 1;
+        constexpr int _Op =
+            (_Bytes ? _SIDD_UBYTE_OPS : _SIDD_UWORD_OPS) | _SIDD_CMP_EQUAL_ANY | _SIDD_LEAST_SIGNIFICANT;
+        constexpr int _Part_size_el = _Bytes ? 16 : 8;

```

Comment:
While these are lower risk because they're `constexpr` (modifiable state would be higher risk), it would still be better to scope these within the conditional statement where they're used.